### PR TITLE
Update README.md for ovm

### PR DESCRIPTION
**Pull request by syuhei176**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,47 @@
...
+We are aiming to general L2 development framework based on Optimistic Virtual Machine.
+The primary goal of this repository is to implement Rust client following OVM standard and to build specific L2 constructions such as Channel, Plasma and Optimistic rollup. Furthermore, this client enables more generalized applications on these constructions.
 
-## Build Source Code
+**This is an experimental software, does not run in a production yet.**
 
-```
-cargo build --release
-```
+## What we do in the repository
```
Comment (syuhei176) [inline README.md]: I will add "Introduction".

Comment (syuhei176) [reply README.md]: added

File: README.md [hunk 2]
```diff
@@ -1,26 +1,53 @@
...
+Testing all crates.
 
 ```
-./target/release/plasma-chamber 
+cargo test --all
 ```
+
+## About us
+
+[Company Site](https://www.cryptoeconomicslab.com)
```
Comment (yuriko627) [inline README.md]: [Corporate Website](https://www.cryptoeconomicslab.com)
[Medium](https://medium.com/cryptoeconomics-lab)

File: README.md [hunk 3]
```diff
@@ -1,26 +1,53 @@
...
-### Run Manually
+### Multi platform
+
+- Run on Linux and Mac
+- Android Integration: https://github.com/cryptoeconomicslab/plasma-android-sdk
+- Browser: TBD
+
+### Smart contracts
+
+OVM core smart contract(Universal Adjudicator contract), predicate and deposit contract is [here](https://github.com/cryptoeconomicslab/ovm-contracts).
```
Comment (yuriko627) [inline README.md]: OVM core smart contract(Universal Adjudication Contract), Predicates, and Deposit And Exit Contract are [here](https://github.com/cryptoeconomicslab/ovm-contracts).

File: README.md [hunk 4]
```diff
@@ -1,26 +1,53 @@
...
+### OVM Client implementation
+
+This repository don't include smart contract, but includes components below.
+
+- The decision mechanism and core deciders
+- Networking utilities for both Layer 1 and Layer 2
+- Plasma and State Channel client implementation
 
-### Run Manually
+### Multi platform
```
Comment (yuriko627) [inline README.md]: #### Multiplatform

Comment (syuhei176) [reply README.md]: changed!

File: README.md (hunk 3 above)
Comment (syuhei176) [reply README.md]: Thanks changed! I modified all points you reviewed!

**Pull request merged**